src/App.js

```diff
@@ -1,13 +1,72 @@
-import React from 'react';
+import React from "react";
+
+import TodoList from "./components/TodoComponents/TodoList";
+
+import TodoForm from "./components/TodoComponents/TodoForm";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: "",
+      completed: false,
+      id: Date.now()
+    };
```

Comment:
Since you are initiating a new Item when `addItem` is invoked you actually don't need to stole the completed and id fields here in state.